packages/cozy-mespapiers-lib/src/components/Search/SearchProvider.jsx

```diff
@@ -1,6 +1,8 @@
 import React, { createContext, useContext, useMemo, useState } from 'react'
+
 import useRealtime from 'cozy-realtime/dist/useRealtime'
 import { useClient } from 'cozy-client'
+
 import { FILES_DOCTYPE, CONTACTS_DOCTYPE } from '../../doctypes'
```

Comment:
OK nice for the move, but now, what we have in the `Search` folder, should not access files outside of this folder (except third party plugins). 

I'll not block this PR for now, but for instance, we should pass all the searchable doctypes to the `SearchProvider`. Like that, the SearchProvider map on them, and connect the realtime by itself. After that, if we want to plug Contacts or else, it's just a props. 

Some for `scannerT` it should be passed on a props like `additionnalTranslation` or else. 

This is something you should take into consideration for the next PR. 🙏

Reply:
le sujet est en standby là, je regarderai quand je reprendrai le sujet 👍

Reply:
corriger dans cette PR https://github.com/cozy/cozy-libs/pull/2044